Resources/views/full/video.html.twig

```diff
@@ -0,0 +1,38 @@
+{% extends "eZDemoBundle::pagelayout.html.twig" %}
+
+{% block content %}
+    {% set videoPath = path( ez_route( 'ez_content_download', { 'content': content, 'fieldIdentifier': 'file' } ) ) %}
+    {% set videoType = content.getField( 'file' ).value.mimeType|default( 'video/mp4' ) %}
+
+    <script>
+        _V_.options.flash.swf = '{{ asset( 'bundles/ezdemo/flash/video-js.swf' ) }}';
+    </script>
+
+    <section class="content-view-full">
+        <article class="class-video">
+            <div class="attribute-header">
+                <h1>{{ ez_content_name( content ) }}</h1>
+            </div>
+
+            <div class="attribute-video">
+                {{ ez_render_field( content, 'file', {
+                    'attr': { 'id': 'video_' ~ content.id, 'class': 'video-js vjs-default-skin', 'data-setup': '{}' },
+                    'parameters': { 'width': '770', 'height': '318', 'videoType': videoType },
+                    'template': 'eZDemoBundle:fields:ezbinaryfile_video.html.twig'
```

Comment:
This is probably offtopic, but why are we using binary field to store a video in a content of type video? Why don't go for a media field instead?